modules/home-manager/globals.nix

```diff
@@ -11,5 +11,7 @@
       default = "teal";
       description = "Global Catppuccin accent";
     };
+
+    enable = lib.mkEnableOption "Catppuccin globally";
```

Comment:
nit: i'd prefer this to go above the other options

Reply:
Fixed